Purchases/Public/RCPurchaserInfo.m

```diff
@@ -82,7 +85,8 @@ - (void)initializePurchasesAndEntitlementsWithSubscriberData:(NSDictionary *)sub
                                                subscriptions:(NSDictionary *)subscriptions {
     NSDictionary<NSString *, NSArray *> *nonSubscriptions = subscriberData[@"non_subscriptions"];
     self.nonConsumablePurchases = [NSSet setWithArray:[nonSubscriptions allKeys]];
-
+    self.nonConsumableTransactionsByProduct = [PurchaserInfoHelper nonConsumableTransactionsMapWithNonSubscriptionsDictionary:subscriberData dateFormatter:dateFormatter];
+    self.nonConsumableTransactionsList = [PurchaserInfoHelper nonConsumableTransactionsListWithNonSubscriptionsTransactionsDictionary:self.nonConsumableTransactionsByProduct];
```

Comment:
Widening from that. How does a list of "non consumable transactions" improve our consumable support?